.idea/Present-App.iml

```diff
@@ -2,7 +2,7 @@
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$" />
-    <orderEntry type="jdk" jdkName="Python 2.7.13 (/usr/bin/python2.7)" jdkType="Python SDK" />
+    <orderEntry type="jdk" jdkName="Python 3.6.3 (/Library/Frameworks/Python.framework/Versions/3.6/bin/python3.6)" jdkType="Python SDK" />
```

Comment:
I don't know why it keeps pushing this file, I thought it was in the gitignore??

Reply:
I configured my default configure thing to be python 3.6 bc it kept saying one wasn't set up and it was irritating. Not sure why it changed it on here tho

Reply:
yeah I know dw.. more of a note to myself..